src/wp-includes/functions.php

```diff
@@ -450,15 +450,21 @@ function number_format_i18n( $number, $decimals = 0 ) {
  */
 function size_format( $bytes, $decimals = 0 ) {
 	$quant = array(
-		'TB' => TB_IN_BYTES,
-		'GB' => GB_IN_BYTES,
-		'MB' => MB_IN_BYTES,
-		'KB' => KB_IN_BYTES,
-		'B'  => 1,
+		/* translators: File size in terabytes. */
```

Comment:
This is no file size, it's the unit. So better something like this:

```suggestion
		/* translators: Unit symbol for terabyte. */
```